### PR TITLE
DOC: Add note about random seeds for unit testing

### PR DESCRIPTION
**Pull request by endolith**

### CODE REVIEW

File: doc/TESTS.rst.txt [hunk 1]
```diff
@@ -374,3 +374,13 @@ at the end of the test run.  Skipped tests are marked as ``'S'`` in
...
 1``).
+
+Tests on random data
+--------------------
+
+Tests on random data are good, but since test failures are meant to
+expose new bugs or regressions, a test that passes most of the time
+but fails occasionally with no code changes is not helpful.
+Make the random data deterministic by setting the random number seed
+before generating it.  Either Python's ``random.seed()`` or Numpy's
```
Comment (charris) [inline doc/TESTS.rst.txt]: "Use either ...". Also `random.seed(your_number_here)` or something similar. `random.seed()` is valid as is, but does not lead to repeatable tests ;)

**Comment by endolith** [comment]
ok, changed

**Comment by charris** [comment]
@endolith Thanks.

But..., It would be best to squash little fixup commits next time around and use the `DOC: ` on the first commit message ;)

**Comment by rgommers** [comment]
nice addition, this often goes wrong. thanks @endolith

**Pull request merged**